regra_de_negocio/gerenciador_turmas.py

```diff
@@ -19,29 +24,21 @@ def editar_turma_svc(id, nome, professor, data_de_inicio):
         return True
     else:
         return False
-
-
 # Parâmetro: um dicionário onde cada turma é um par chave-valor
 # Retorna:
 #   True se a operação for bem sucedida
-def _salvar_turmas(turmas):
-    dados = json.dumps(turmas, indent=4)
-    with open("dados/turmas.json", "w", encoding="utf-8") as arquivo:
-        arquivo.write(dados)
-    return True
-
-
-def busca_grupos():
-    with open("dados/grupos.json", "r", encoding="utf-8") as f:
-        grupos_Data = json.load(f)
-    return grupos_Data
-
 
 # Função para criar uma nova turma
-def criacao_turma(dados_nova_turma):
+def criacao_turma(dados_nova_turma, grupos):
+    """
+    Dados da nova turma vem no seguinte formato:
+    nome: nome da turma
+    professor: nome professor
+    data_de_inicio: data selecionada
+    grupos: uma lista dos grupos que foram selecionados para perteceram a essa turma
+    """
     dados_nova_turma_json = json.loads(dados_nova_turma)
     turmas = busca_turmas()
-    grupos = busca_grupos()
 
     turma_novo_id = str(len(turmas) + 1)
```

Comment:
Não, isso vai dá pau! Pq imagina, a gente tem a turma 0, 1, 2, 3, 4 Porém eu deleto o 1 e 3! Quando for criar turma, len(turmas) vai ser igual a 3, se vc adiciona 1, fica 4. Vamos ter então dois ids repetidos. Você pode buscar qual é o ID mais alto e adicionar 1.